README.md

```diff
@@ -287,6 +287,37 @@ Kubeclient::Client.new(
 ```
 
 
+#### Amazon EKS Credentials
+
+On Amazon EKS, when using aws-iam-authentication, with [credentials](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/Credentials.html),
+kubeclient can use `aws-sdk-core` gem to retrieve a token to authenticate against EKS.
+
+This requires a set of gems which are _not_ included in
+`kubeclient` dependencies (`aws-sdk-core` and `aws-sigv4`) so you should add them to your bundle.
+
+To obtain a token:
+
+```ruby
+require 'aws-sdk-core'
```

Comment:
Can this be omitted?  Its required in token().

Reply:
i've removed it from token but will be required for anyone constructing the Aws::Credentials object